src/utils/useConnectionOptions/useConnectionOptions.ts

```diff
@@ -37,13 +37,21 @@ export default function useConnectionOptions() {
     // their individual bandwidth constraints. Simulcast should be disabled if
     // you are using Peer-to-Peer or 'Go' Rooms.
     preferredVideoCodecs: [{ codec: 'VP8', simulcast: roomType !== 'peer-to-peer' && roomType !== 'go' }],
+
+    //@ts-ignore - Internal use only. This property is not exposed in type definitions.
+    environment: process.env.REACT_APP_TWILIO_ENVIRONMENT,
```

Comment:
we should include this only when `process.env.REACT_APP_TWILIO_ENVIRONMENT !== 'prod` ` so that apps that base off react app wont pass this parameter.

Reply:
It seems to work fine when `REACT_APP_TWILIO_ENVIRONMENT=prod`. I'm not sure that this is needed.

Setting `environment: 'prod'` here is the same thing the SDK does [here](https://github.com/twilio/twilio-video.js/blob/master/lib/connect.js#L212). So I don't think it causes any issues.

Reply:
sure, it does not cause any issues, but its not one of the documented parameter. I would not recommend that our customers pass it.  I would do  something like:

```
  // Internal use only - remove this code block
  if (environment !== 'prod') {
    //@ts-ignore - Internal use only. This property is not exposed in type definitions.
    connectionOptions!.environment = environment;
    if (environment === 'dev') {
      //@ts-ignore - Internal use only. This property is not exposed in type definitions.
      connectionOptions!.wsServer = 'wss://us2.vss.dev.twilio.com/signaling';
    }
  }
```

Reply:
> I would not recommend that our customers pass it

I agree. I wouldn't customers using `REACT_APP_TWILIO_ENVIRONMENT` either, which is why I decided to leave it out of the [.env.example](https://github.com/twilio/twilio-video-app-react/blob/master/.env.example) file. I think this is enough to keep customers away from it. It will be an undocumented parameter, just like `connectOptions.environment`.

I don't think we need to add another if statement to prevent `environment: 'prod'` from being set, especially since that what the SDK sets it to by default.

Reply:
sure.